### ISSUE TITLE
Build is broken on Windows

### ISSUE THREAD

**Issue by FaustinCarter** [conda/constructor#429]
A quick look at the CI tests shows that the latest several runs are all broken on Windows. I've confirmed this on my own Windows box. The issues appears to stem from running constructor on `examples/miniconda`. There is some sort of duplicate files error.

I have confirmed that building is _also_ broken on at least one previously known good commit (the release commit for 3.2.0). I suspect this is due to a change in a dependency somewhere, but haven't had time to track it down.

To reproduce the problem (on Windows), create a new environment, install constructor, clone the constructor repo, and run `constructor examples/miniconda`.

**Comment by FaustinCarter** [comment]
I've tracked this down to an issue with the packages in http://repo.anaconda.com/pkgs/main/. Switching the source URL to https://anaconda.org/conda-forge/ solves the build errors across the board.

**Comment by FaustinCarter** [comment]
This issue is definitely relevant: #336. Looks like several new Windows packages are including files they shouldn't? @mingwandroid Can you look into this since you managed to fix it last time this cropped up? [Logfiles in the failed Windows CI jobs](https://github.com/conda/constructor/actions) have the full logs, but I summarize here (heading is the python version in the actions matrix):

The offending packages are same in every case:

- win_inet_pton-1.1.0-py39haa95532_0.conda
- six-1.15.0-py39haa95532_0.conda
- pysocks-1.7.1-py39haa95532_0.conda

In test runs on my own machine trying to run `constructor` on the `examples/miniconda`, I've had other packages generate the error too (including `brotlipy`, `cffi`, and `cryptography`) when changing my build env up slightly (i.e. py38, conda 4.9.0, etc).

## py27
```
Error: File 'Lib/site-packages/wheel/vendored/packaging/__pycache__/__init__.cpython-39.pyc' found in multiple packages: win_inet_pton-1.1.0-py39haa95532_0.conda, six-1.15.0-py39haa95532_0.conda, pysocks-1.7.1-py39haa95532_0.conda
```

## py36
```
Error: File 'Lib/site-packages/wheel/__pycache__/__init__.cpython-39.pyc' found in multiple packages: win_inet_pton-1.1.0-py39haa95532_0.conda, pysocks-1.7.1-py39haa95532_0.conda, six-1.15.0-py39haa95532_0.conda
```

## py37 
```
Error: File 'Lib/site-packages/wheel/__pycache__/__init__.cpython-39.pyc' found in multiple packages: six-1.15.0-py39haa95532_0.conda, win_inet_pton-1.1.0-py39haa95532_0.conda, pysocks-1.7.1-py39haa95532_0.conda
```

## py38
```
Error: File 'Lib/site-packages/wheel/__pycache__/__init__.cpython-39.pyc' found in multiple packages: six-1.15.0-py39haa95532_0.conda, pysocks-1.7.1-py39haa95532_0.conda, win_inet_pton-1.1.0-py39haa95532_0.conda
```

**Comment by FaustinCarter** [comment]
I had some time to look into this a bit more. All of the different builds for the 1.15.0 version of the `six` package on `https://repo.anaconda.com/pkgs/main/win-64` contain a bunch of extra stuff in their `/Lib/site-packages` directory, one of which is the file that `constructor` is complaining about. I'm not sure why all this stuff is getting included. Versions prior to 1.15.0 do not have all this extra stuff.

I wonder if something changed in the build process over at Anaconda? The `conda-forge` packages do not have this issue.

**Comment by FaustinCarter** [comment]
This was closed by #435.
